requirements.txt

```diff
@@ -7,9 +6,11 @@ SQLAlchemy==0.9.8
 Sphinx==1.2.3
 Werkzeug==0.9.6
 alembic==0.8.2
+backtranslate==0.0.4
```

Comment:
It is not required, there were no changes in the interface (except that `backtranslate` does returns a `dict` instead of a `defaultdict`). It would not hurt either.